xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/lightbox.js

```diff
@@ -301,13 +305,17 @@ define('xwiki-lightbox', [
         $('#lightboxDownload').attr('download', imageData.fileName);
       }
     };
-    myOpenLightbox = gallery(media, options);
+    _openedLightbox = gallery(_slidesData, options);
   };
 
   /**
    * Initialize the lightbox functionality for a set of images.
    */
   var initLightboxFunctionality = function() {
+    // The lightbox will be added to xwikicontent images that don't explicitly disable it.
+    _lightboxImages = $('#xwikicontent img')
+      .filter((i, img) => $(img.closest('[data-xwiki-lightbox]')).data('xwiki-lightbox') != false);
```

Comment:
I'm not talking about the name of the HTML attribute. Here I'm talking about how to specify the name in jQuery. `data('xwikiLightbox')` also reads the `data-xwiki-lightbox` attribute, see the [documentation of `data(key)`](https://api.jquery.com/data/#data2).

Reply:
This camelCase variant is used also, e.g., in https://github.com/xwiki/xwiki-platform/blob/05f0ba0694a3e5e4395f493c3388ac152d218ad4/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/syntaxPicker.js#L197 - the attribute is `data-saved-syntax` here.

Reply:
I have a slightly preference for using camel-case when accessing the data (not the attribute) but I'm not sure it deserves a best practice rule (since jQuery supports both).